### DISCUSSION TITLE
Need Guidance on ComfyUI API for SDXL 0.9

### DISCUSSION THREAD

**Discussion by tykim9999** [Comfy-Org/ComfyUI#909] [Q&A] [answered]
Hello, I'm a beginner trying to navigate through the ComfyUI API for SDXL 0.9. I'm having a hard time understanding how the API functions and how to effectively use it in my project. If anyone could share a detailed guide, prompt, or any resource that can make this easier to understand, I would greatly appreciate it. Thanks in advance for your help!

**Comment by tykim9999** [comment]
I have found a solution to the problem. For anyone else who may encounter the same issue, here's what I did:

"Check EXIF metadata"

Please feel free to ask if you have any questions or need further clarification. I hope this helps someone else as well!

**Comment by comfyanonymous** [comment, marked as answer]
if you update to the latest you can "enable dev mode options" in the settings (gear beside the "Queue Size: ") this will enable a button on the UI to save workflows in api format.

You can then use these api jsons in these script examples: https://github.com/comfyanonymous/ComfyUI/tree/master/script_examples
